devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -371,21 +371,49 @@ def getTemperature(value) {
 
 	def refresh() {
 		log.debug "Refreshing Values "
-		def refreshCmds = [
         
-        /* sensitivity - default value (8) */
+        def refreshCmds = [
+	            "zcl mfg-code ${manufacturerCode}", "delay 200",
```

Comment:
Since the `zcl mfg-code` is part of the WriteAttribute transaction including the `zcl global write` and the `send` it is OK and preferable to put it in the if/else block below even though it means there is a little bit of duplicate code.